app/Providers/AuthServiceProvider.php

```diff
@@ -28,11 +28,15 @@ public function boot(GateContract $gate)
     {
         $this->registerPolicies($gate);
 
-        // assign new users to the latest group (for now)
-        $defaultGroup = Semester::orderBy('id', 'desc')->first();
+        $pdo = new \PDO(getenv('DB_CONNECTION').':host='.getenv('DB_HOST').';dbname='.getenv('DB_DATABASE'), getenv('DB_USERNAME'), getenv('DB_PASSWORD'));
```

Comment:
This should probably just be a try/catch block, we don't want to introduce layers for the connection like this